spec/features/user_visits_book_show_spec.rb

```diff
@@ -0,0 +1,67 @@
+describe "User visits book show page" do
+  context "sees book title and reviews" do
+    before(:all) do
+      @book = Book.create(title: "Bible")
+      @user_1 = User.create(name: "KenM")
+      @user_2 = User.create(name: "Max")
+
+      @review_1 = @book.reviews.create(body: "Best story book ever",
+                                   user_id: @user_1.id,
+                                   rating: 5)
+      @review_2 = @book.reviews.create(body: "Does not give water to wine recipe",
+                                     user_id: @user_2.id,
+                                     rating: 1)
+    end
+
+    it "including review: body, user, rating" do
+      visit book_path(@book)
+
+      expect(current_path).to eq("/books/#{@book.id}")
```

Comment:
Since you've just told your test to visit this page, this line is testing against the capybara visit method. I'd use an expect(current_path) assertion if I had created a link or submitted a form to check that I've done it right where here you are testing a method someone else wrote.